101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "certificatePrefixName": {
+      "type": "string"
+    },
+    "certificatePfxBase64": {
+      "type": "securestring"
+    },
+    "certificatePfxPassword": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "storagename": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "batchname": "[concat('batch', uniqueString(resourceGroup().id))]",
+    "locationname": "[resourceGroup().location]",
+    "keyvaultname": "[concat('keyvault', uniqueString(resourceGroup().id))]",
+    "tenantId": "[subscription().tenantId]",
+    "certificateName": "[concat(parameters('certificatePrefixName'), uniqueString(resourceGroup().id))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-08-01",
```

Comment:
update apiVersion

Reply:
Done. updated to  "apiVersion": "2018-02-01",